src/components/enterprise-user-subsidy/UserSubsidy.jsx

```diff
@@ -32,9 +33,9 @@ const UserSubsidy = ({ children }) => {
         hasAccessToPortal = subscriptionLicense?.status === LICENSE_STATUS.ACTIVATED;
       }
 
-      return { hasAccessToPortal, subscriptionLicense };
+      return { hasAccessToPortal, subscriptionLicense, offers };
```

Comment:
@raq929 I think the logic around `hasAccessToPortal` will need to change to support the case where my enterprise has a subscription but I do not have a license, yet I have a code assigned to me.

With the logic as is, if my enterprise has a subscription but I don't have a license, we prevent the user from viewing any other page in the LP except for the Dashboard page (see the `SubscriptionSubsidy` component).

But since I have a code assigned to me in this case, I should still be able to view the search/course pages to redeem my code, right?

Reply:
Yeah, that will definitely need to change. But in the next PR. I'm trying not to let the scope creep too far on this one, and have it be just the refactor. ;)